### DISCUSSION TITLE
Cross-domain settings do not take effect

### DISCUSSION THREAD

**Discussion by sswfive** [fastapi/fastapi#8619] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
def register_cors_middleware(app: FastAPI) -> None:
    logger.info("register cors middleware")
    app.add_middleware(
        CORSMiddleware,
        allow_origins=[],
        # allow_origins=["http://127.0.0.1:7878"],
        allow_credentials=True,
        allow_methods=['POST'],
        allow_headers=['*'],
    )
```

### Description

When I configure the allow_origins parameter to be empty, or specify the ip, the server side is not intercepted, and the request can still be made.

### Operating System

Linux

### FastAPI Version

fastapi=0.75.0

### Python Version

Python 3.9.10

**Comment by YuriiMotov** [comment, marked as answer]
Everything works as expected in current version (0.115.13).

The problem was probably in how the request was sent
